relm4-macros/src/attrs.rs

```diff
@@ -1,31 +1,125 @@
 use proc_macro2::TokenStream as TokenStream2;
 use quote::ToTokens;
-use syn::{
-    parse::{Parse, ParseStream},
-    Result, Visibility,
-};
+use syn::Ident;
+use syn::Result;
+use syn::Token;
+use syn::Visibility;
+use syn::parse::Parse;
+use syn::parse::ParseStream;
 
 #[derive(Debug)]
-pub struct Attrs {
+pub struct WidgetVisibility {
```

Comment:
I don't think this struct is necessary. `Option<Visibility>` already implements ToTokens.

Reply:
Removed